playbooks/appsemblerPlaybooks/eucalyptus_enterprise_deploy.yml

```diff
@@ -0,0 +1,105 @@
+---
+
+# Appsembler pro tier deployment
+# Docs: https://github.com/appsembler/openedx-docs/blob/master/openedx/pro_tier_deployment.md
+
+- name: Install Mongo Replica Set
+  hosts: "mongo-servers"
+  sudo: True
+  gather_facts: True
+  vars:
+    BACKUPS_MONGO: False
+    BACKUPS_MYSQL: False
+    COMMON_ENABLE_BACKUPS: False
+  roles:
+    - { role: swapfile, SWAPFILE_SIZE: "2GB" }
+    - mongo_3_0
+
+
+- name: Install Mongo Backups
+  hosts: "mongo-arbiter"
+  sudo: True
+  gather_facts: True
+  vars:
+    BACKUPS_MONGO: True
+    BACKUPS_MYSQL: False
+    COMMON_ENABLE_BACKUPS: True
+  roles:
+    - backups
+
+
+- name: Install Needed Packages On Services Server
+  hosts: "services-server"
+  sudo: True
+  gather_facts: True
+  roles:
+    - { role: swapfile, SWAPFILE_SIZE: "4GB" }
+    - role: sudo
```

Comment:
@mrgnr I think so, in [amc.yml playbook](https://github.com/appsembler/configuration/blob/appsembler/eucalyptus/develop/playbooks/amc.yml#L18) is used in this way, I did not tested yet without `- role:` but I think it will fail, maybe @denibertovic can confirm, is the author of the role.